src/format_csharp.js

```diff
@@ -152,7 +152,7 @@ class CSharpFormatter {
         }
         switch (fullName(member)) {
           case 'BrowserContext.addCookies.cookies': return '`Cookie`';
-          case 'BrowserContext.cookies': return '`Cookie`';
+          case 'BrowserContext.cookies': return '`BrowserContextCookiesResult`';
```

Comment:
This mirrors https://github.com/microsoft/playwright/blob/79c4505433c30e63588acc0a5b33d572baa73488/utils/doclint/generateDotnetApi.js#L427